packages/local_auth/local_auth_ios/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.1
```

Comment:
In case anyone (including me when I forget) is confused by this later: this is a minor version bump rather than a major version bump even though it lists a breaking change because we haven't started using this package yet. Since nobody is using 1.0.0, there's nobody to break.